articles/cms/wordpress/extending.md

```diff
@@ -0,0 +1,230 @@
+---
+description: How to extend the Login by Auth0 WordPress Plugin with hooks, filters, and functions.
+---
+
+# Extending Login by Auth0 WordPress Plugin
+
+We're happy to review and approve new filters and actions that help you integrate even further in this plugin. Please
+ see the Contributing section on the [GitHub repo readme for this plugin](https://github.com/auth0/wp-auth0/blob/master/README.md).
+
+## Actions
+
+Actions in WordPress run custom code at specific points during processing. [Learn more about actions here](https://developer.wordpress.org/plugins/hooks/actions/). 
+
+### `auth0_before_login`
```

Comment:
For headings that have function names or variables or whatever else we normally don't do the backticks, sometimes formatting can go kind of wonky. Same with the others below.

Reply:
@jeffreylees - For these, I thought it would be handy to have an anchor link and have them stand out a bit. Formatting seems to work:

https://auth0-docs-content-pr-5707.herokuapp.com/docs/cms/wordpress/extending

Any other ideas?

Reply:
I'll have to point this out as well since it is not consistent with the formatting we follow at all docs. We don't use backtips on headers (it doesn't look like a header any longer but some hybrid) so in order to be consistent with the rest of the documentation you have to remove them.